### ISSUE TITLE
from pyexiv2 import Image

### ISSUE THREAD

**Issue by chellChang** [LeoHsiao1/pyexiv2#68]
Traceback (most recent call last):
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 855, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/opt/homebrew/lib/python3.9/site-packages/pyexiv2/__init__.py", line 6, in <module>
    from .core import *
  File "/opt/homebrew/lib/python3.9/site-packages/pyexiv2/core.py", line 1, in <module>
    from .lib import exiv2api
  File "/opt/homebrew/lib/python3.9/site-packages/pyexiv2/lib/__init__.py", line 25, in <module>
    ctypes.CDLL(os.path.join(lib_dir, 'libexiv2.dylib'))
  File "/opt/homebrew/Cellar/python@3.9/3.9.5/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ctypes/__init__.py", line 374, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: dlopen(/opt/homebrew/lib/python3.9/site-packages/pyexiv2/lib/libexiv2.dylib, 6): image not found

**Comment by LeoHsiao1** [comment]
Thank you for your report.This looks like a compatibility issue.

Let's debug it:
1. Which version of Pyexiv2 do you use?
2. Which version of MacOS do you use?
3. Can you manually execute the following Python code?
    ```py
    import os
    import ctypes
    path = '/opt/homebrew/lib/python3.9/site-packages/pyexiv2/lib/libexiv2.dylib'
    os.path.isfile(path)
    ctypes.CDLL(path)
    ```

**Comment by manuelliebchen** [comment]
I have the same issue:
1. Version of pyexiv2: 2.6.0
2. macOs Big Sur 11.4, Apple M1
3. When I execute the code the same error message as above appearse.

**Comment by LeoHsiao1** [comment]
> I have the same issue:
> 
> 1. Version of pyexiv2: 2.6.0
> 2. macOs Big Sur 11.4, Apple M1
> 3. When I execute the code the same error message as above appearse.

Thank you for your report. It seems that there is a problem with version 2.6.0.
You can use version 2.5.0 first.I'll be testing MacOS in a virtual machine these days.

**Comment by manuelliebchen** [comment]
Allright, that works, thanks :)

**Comment by LeoHsiao1** [comment]
Hi, I just released [v2.6.2](https://github.com/LeoHsiao1/pyexiv2/releases/tag/v2.6.2). Please try it on MacOS 11 .
